server/hedley/modules/custom/hedley_restful/plugins/restful/node/HedleyRestfulSync.class.php

```diff
@@ -515,24 +515,27 @@ public function handleChanges() {
     catch (Exception $e) {
       $transaction->rollback();
 
-      $m1 = '[MAIN] - ';
-      $m2 = '[DATA] - ';
+      $main = '[MAIN] -';
+      $data = '[DATA] -';
       foreach ($item as $k => $v) {
-        $m1 .= "  $k: $v  ||| ";
+        if ($k == 'data') {
+          continue;
+        }
+        $main .= "  $k: $v";
       }
-
       foreach ($item['data'] as $k => $v) {
         $value = is_array($v) ? implode(', ', $v) : $v;
-
-        $m2 .= "  $k: $value";
+        $data .= "  $k: $value";
       }
 
-      watchdog('debug', $m1, [], WATCHDOG_ERROR);
-      watchdog('debug', $m2, [], WATCHDOG_ERROR);
+      $details = $main . PHP_EOL . PHP_EOL . $data;
+      watchdog('debug', $details, [], WATCHDOG_ERROR);
```

Comment:
```suggestion
      watchdog('hedley_restful', $details, [], WATCHDOG_ERROR);
```

perhaps, not sure if this `debug` is used at other places too consistently or not